include/mapbox/vector_tile.hpp

```diff
@@ -40,7 +40,7 @@ class feature {
     feature(protozero::data_view const&, layer const&);
 
     GeomType getType() const { return type; }
-    mapbox::feature::value getValue(std::string const&) const;
+    mapbox::feature::value getValue(std::string const&, std::string* message = nullptr) const;
```

Comment:
is `message` nullable?  if not, better to use reference

Reply:
Yes, I want to make it nullable so that it is backward compatible and won't break existing codes that use this API.

Reply:
@zmiao, could you add a description for the newly added `out` parameter? How it is supposed to be used, etc.